### ISSUE TITLE
Show Run/Debug in same line of @Test

### ISSUE THREAD

**Issue by gayanper** [angelozerr/jdt-codemining#30]
By showing the Run | Debug on the same line next to @Test would reduce the line hence reducing the chance for performance issues with large test files.

**Comment by vogella** [comment]
I personally like the current behavior. Too much configuration is also a bad thing. I think is also hard to follow the logic that placing the annotations somewhere else, will reduce  the chance for performance issues with large test files. 

I personally would put this into WONTFIX
